packages/fiori/src/TimelineItem.ts

```diff
@@ -89,8 +89,21 @@ class TimelineItem extends UI5Element implements ITimelineItem {
 	@property()
 	subtitleText?: string;
 
+	/**
+	 * Defines the content of the `ui5-timeline-item`.
+	 * @public
+	 */
+	@slot({ type: HTMLElement, "default": true })
+	content!: Array<HTMLElement>;
```

Comment:
now, when the default slot has accessor we can replace previous expressions where  innerHTML was uused to content.length